src/main/java/trade/wayruha/mexc/enums/OrderSide.java

```diff
@@ -26,4 +26,9 @@ public static OrderSide findByAlias(int aliasNumber){
         return Arrays.stream(OrderSide.values()).parallel().filter(s-> s.getAliasNumber() == aliasNumber)
                 .findFirst().orElse(null);
     }
+
+    public static OrderSide findByName(String name){
+        return Arrays.stream(OrderSide.values()).parallel().filter(s-> s.name().equals(name))
+                .findFirst().orElse(null);
+    }
 }
```

Comment:
how is it different from Enum.valueOf(String)?

Reply:
almost nothing, but "Enum.valueOf" push Exception if not found